### PR TITLE
[testbed][KVM] Fix minor u20 issues

### PR DESCRIPTION
**Pull request by myronsosyak**
### Description of PR

Summary: Fix minor issues faced during virtual testbed bring up on u20.04
Fixes # (issue)

### Type of change

- [x] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

#### What is the motivation for this PR?
I have tried to bring up virtual testbed on clean u20.04 server and faced some issues, mostly python related
#### How did you do it?
- install dependencies in proper way
- fix kikstart.py

#### How did you verify/test it?
Follow https://github.com/Azure/sonic-mgmt/blob/master/docs/testbed/README.testbed.VsSetup.md to bring up vtestbed
#### Any platform specific information?
N/A

### CODE REVIEW

**Comment by myronsosyak** [comment]
@wangxin Please review

File: setup-container.sh [hunk 1]
```diff
@@ -46,7 +46,8 @@ GROUPNAME=$USER
 USERNAME=$USER
 EOF
 
-    j2 -o $tmpdir/Dockerfile $tmpdir/Dockerfile.j2 $tmpdir/data.env
+    j2 -ozzz $tmpdir/Dockerfile $tmpdir/Dockerfile.j2 $tmpdir/data.env || \
+    j2 $tmpdir/Dockerfile.j2 $tmpdir/data.env > $tmpdir/Dockerfile
```
Comment (lguohan) [inline setup-container.sh:50]: what is the purpose of this change?

Comment (wangxin) [reply setup-container.sh:50]: Same concern here. Tried `j2 -ozzz $tmpdir/Dockerfile $tmpdir/Dockerfile.j2 $tmpdir/data.env`, it failed with error like below:
```
$ j2 -ozzz $tmpdir/Dockerfile $tmpdir/Dockerfile.j2 $tmpdir/data.env
usage: j2 [-h] [-v] [-f {?,json,ini,env}] [-e VAR]
          [--filters python-file [python-file ...]]
          [--tests python-file [python-file ...]] [--customize python-file.py]
          [--undefined] [-o outfile]
          template [data]
j2: error: unrecognized arguments: /tmp/tmp.3LbDNSU8Uh/data.env
```

Comment (myronsosyak) [reply setup-container.sh]: My apologies. It is not needed. Removed

**Review by lguohan — APPROVED** [review]

**Review by wangxin — APPROVED** [review]
